solang-parser/src/pt.rs

```diff
@@ -55,7 +55,7 @@ pub struct SourceUnit(pub Vec<SourceUnitPart>);
 #[derive(Debug, PartialEq)]
 pub enum SourceUnitPart {
     ContractDefinition(Box<ContractDefinition>),
-    PragmaDirective(Vec<DocComment>, Identifier, StringLiteral),
+    PragmaDirective(Vec<DocComment>, Identifier, StringLiteral, Loc),
```

Comment:
minor: `Loc` is usually the first field of enum variant
```suggestion
    PragmaDirective(Loc, Vec<DocComment>, Identifier, StringLiteral),
```

Reply:
thanks for the comment, fixed